scitt/create_detatched_signed_statement.py

```diff
@@ -0,0 +1,217 @@
+""" Module for creating a SCITT signed statement with a detached payload"""
```

Comment:
Suggest renaming: `create-hashed-signed-statement.py` as it's not "detached" from a COSE draft perspective.

Reply:
good suggestion will fix!

Reply:
done!